lua/nordic/groups/integrations.lua

```diff
@@ -97,39 +97,83 @@ function M.get_groups()
     G.MiniStatuslineFileinfo = { bg = LC.normal.b.bg, fg = LC.normal.b.fg }
     G.MiniStatuslineInactive = { bg = LC.inactive.a.bg, fg = LC.inactive.a.fg, bold = LC.inactive.a.gui == 'bold' }
 
+    -- Tree Groups
+    G.TreeNormal = { fg = C.fg, bg = C.bg }
+    G.TreeNormalNC = { fg = C.fg, bg = C.bg }
+    G.TreeRootFolder = { fg = C.gray4 }
+    G.TreeRootName = { fg = C.white2 }
```

Comment:
This is too close to TreeNormal for me to tell the difference. It just looks a little off. However, that might just be a me problem.

Reply:
idk I was just based on how the "nord" theme managed their colors

Reply:
also if you have something in mind for this part tell me and i'll change it

Reply:
I am thinking it doesn't need it's own color. It's not like it's hard to identify (always the top one).

Reply:
you talking about the TreeNormal or TreeRootName?

Reply:
> you talking about the TreeNormal or TreeRootName?

TreeRootName

Reply:
I think I like that best. What about just making it `G.TreeRootName = G.TreeNormal`

Reply:
done

Reply:
@5-pebbles is there a way to test out these changes I made before merging them to main. I mean I tried this in my config but how do I try the theme as a "whole" with the new changes?

Reply:
> @5-pebbles is there a way to test out these changes I made before merging them to main. I mean I tried this in my config but how do I try the theme as a "whole" with the new changes?

We have automated tests which you should be able to run locally with `nvim --headless -c "lua require('nordic.tests').run_tests()" -c 'q' 2>&1` those are also automatically ran in a VM when you push changes to PR.

Reply:
That doesn't necessarily mean everything is working it just means you haven't broken anything in a way I have already thought of...

Reply:
i pushed so whenever y'all can run the tests and see if it works

Reply:
The tests didn't get it but there is a bug and it's my fault...
![image](https://github.com/user-attachments/assets/30d8ea4b-6d13-43fb-9607-5eedc03cb5f6)

I was wrong when I said the `= G.TreeNormal` thing use this instead: `    G.TreeRootName = { fg = C.fg }
`

Reply:
Apparently that overrides the selected background...

Reply:
done